workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/src/routes/attachment-file-uploader/components/AttachmentUploader/index.jsx

```diff
@@ -0,0 +1,182 @@
+import i18n from '~/common/I18n';
+import AttachmentComment from './components/AttachmentComment';
+import DropzoneUpload from './components/DropzoneUpload';
+import { filesize } from 'filesize';
+import { useState, useEffect } from 'react';
+import FileList from './components/FileList';
+import { Button as ClayButton } from '@clayui/core';
+
+const AttachmentUploader = () => {
+	const [fileAttached, setFileAttached] = useState(null);
+	const [isCheckboxChecked, setIsCheckboxChecked] = useState(false);
+	const [attachmentComment, setAttachmentComment] = useState('');
+	const [ticketAttachmentId, setTicketAttachmentId] = useState(null);
+	const [gcsSessionURL, setGcsSessionURL] = useState('');
+
+	const zendeskTicketId = 378777;
```

Comment:
I think this `zendeskTicketId` shouldn't be hardcoded like this?

Also, can we remove the `zendesk` reference from these variables so we have less to change when/if we migrate to JSM? Can rename all these variables to just like `ticketId` so it stays relevant for both zendesk and jira?

Reply:
The “ticketId” variable will come from parameters in the URL, the behavior of the URL is in another ticket that hasn't been started yet, so I was instructed to mock the value.

About the name change, I'll rename it.

Reply:
Thanks for the clarification, sounds good to mock for now until the other ticket is completed.